packages/tooltips/package.json

```diff
@@ -20,10 +20,10 @@
   },
   "dependencies": {
     "@zendeskgarden/container-tooltip": "^0.2.4",
-    "@zendeskgarden/container-utilities": "^0.1.2",
+    "@zendeskgarden/container-utilities": "^0.3.0",
     "@zendeskgarden/react-selection": "^6.5.0",
     "classnames": "^2.2.5",
-    "react-popper": "0.8.2"
+    "react-popper": "0.10.4"
```

Comment:
I think this needs to be changed to hat notation: `^0.10.0`.

Reply:
Correct. Updated.